examples/basic-http.yaml

```diff
@@ -25,7 +24,7 @@ spec:
         matchLabels:
           app: foo
       namespaces:
-        from: "All"
+        from: "SameNamespace"
```

Comment:
This was simplified because the route and gateway are being installed in the same namespace in this example.